src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -14,12 +14,13 @@ public sealed class PaxTarEntry : PosixTarEntry
     {
         private ReadOnlyDictionary<string, string>? _readOnlyExtendedAttributes;
 
-        // Constructor used when reading an existing archive.
+        // Constructor called when reading a TarEntry from a TarReader.
         internal PaxTarEntry(TarHeader header, TarReader readerOfOrigin)
-            : base(header, readerOfOrigin)
+            : base(header, readerOfOrigin, TarEntryFormat.Pax)
         {
             _header._extendedAttributes ??= new Dictionary<string, string>();
-            _readOnlyExtendedAttributes = null;
+
+            AddNewAccessAndChangeTimestampsIfNotExist();
```

Comment:
I should remove this call because if the entry is coming from an existing reader, I should not edit it.